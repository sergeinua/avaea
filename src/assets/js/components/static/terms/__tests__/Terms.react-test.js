import React from 'react';
import StaticTerms from '../Terms';
import renderer from 'react-test-renderer';


test('StaticTerms component test', () => {
  var mockUser = {email: "test@gmail.com", id: 58};
  var component = renderer.create(
    <StaticTerms user={mockUser} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});
