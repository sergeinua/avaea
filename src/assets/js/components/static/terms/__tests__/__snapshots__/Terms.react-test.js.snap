exports[`test StaticTerms component test 1`] = `
<div>
  <div
    className="content static terms textly">
    <h1>
      Terms & Conditions
    </h1>
    <div
      className="updated">
      Terms of Service
    </div>
    <div
      className="copy intro">
      Here\'s what you\'re agreeing to by accessing or using OnVoya, Inc.\'s websites and its related mobile apps (collectively, the \"OnVoya Service\"):
    </div>
    <div
      className="copy intro">
      The OnVoya Service is owned and operated by OnVoya, Inc. (\"OnVoya,\" \"we\"). These Terms of Service are a binding legal agreement between you and OnVoya regarding your use of the OnVoya Service (together with the Privacy Policy and the Mobile Terms where applicable, the \"Agreement\"). If you do not accept these terms, do not use the OnVoya Service.
    </div>
    <div
      className="sub-copy">
      We reserve the right to modify or replace these Terms of Service at any time and in our sole discretion. When we do, we will update the last revised date at the top of this Agreement. Any changes will be effective immediately upon posting the revised version (or such later effective date as may be indicated at the top of the revised Terms of Service). It is your responsibility to regularly check the OnVoya website to determine if there have been changes. Your continued use of the OnVoya Service following the posting of any changes to the Terms of Service will constitute your acceptance of any such changes. If you do not agree to the changes, you must stop using the OnVoya Service.
    </div>
    <div
      className="subti">
      Your Responsibilities
    </div>
    <div
      className="sub-copy">
      The OnVoya Service is for people trying to find and book travel information for themselves, their families, their coworkers, and their friends, and OnVoya grants you a limited and revocable right to use the Service for those purposes. You represent that all information you supply on the Service is true, accurate, current and complete. If you book travel found on OnVoya on another person\'s behalf, you represent that you are authorized to make that booking, and you agree to inform that person about all terms, conditions, rules, and restrictions applicable to that booking.
    </div>
    <div
      className="sub-copy">
      Without separate, written permission from OnVoya in advance, you may not: (i) reuse or \"scrape\" OnVoya\'s data for use in another service or website, (ii) attempt to circumvent any controls or limitations OnVoya places on your ability to access OnVoya or information on the OnVoya Service, including by means of robot exclusion headers, (iii) use any bots, scrapers, brute-force tools, or other automated methods for accessing, slowing down, or disabling the OnVoya Service or otherwise interfering with the proper function of the website, (iv) \"frame,\" \"mirror,\" or otherwise incorporate any part of the OnVoya Service into any other website or service, or (v) take any action that imposes, or may impose (as determined at our discretion), an unreasonable or disproportionately large load on the OnVoya Service.
    </div>
    <div
      className="subti">
      Intellectual Property
    </div>
    <div
      className="sub-copy">
      The OnVoya Service, and all information, data and other content and materials available on the OnVoya Service, including, but not limited to, the OnVoya logo and all designs, text, documents, graphics, software, sounds files, other files and the selection and arrangement thereof are the proprietary property of OnVoya and its suppliers and licensors and are protected by U.S. and international intellectual property laws. OnVoya and its suppliers and licensors reserve all right, title, and interest, including all intellectual property rights, in and to all such information, data, and other content and materials.
    </div>
    <div
      className="sub-copy">
      ONVOYA and the OnVoya logo are trademarks of OnVoya, and may not be copied, imitated or used, in whole or in part, without the prior written permission of OnVoya. All other trademarks, product names and company names or logos mentioned on the OnVoya website are property of their respective owners. You understand that reference to any products, services, processes or other information, by trade name, trademark, manufacturer, supplier or otherwise, does not constitute or imply endorsement, sponsorship, or recommendation thereof by us, or vice versa.
    </div>
    <div
      className="subti">
      Pre-Release Services
    </div>
    <div
      className="sub-copy">
      Subject to these Terms of Service, OnVoya may make beta and other pre- release services, and related documentation, materials, and information (collectively, the \"Pre-Release Services\") available to you from time to time for the purpose of providing OnVoya with feedback on the quality and usability of the Pre-Release Services. You understand and agree that your use of the Pre-Release Services is solely for testing and evaluation purposes in connection with the OnVoya Service and will not be used for any commercial purposes. During your participation in the Pre-Release Services, OnVoya is not obligated to provide you with any maintenance, technical or other support for the Pre-Release Services. You acknowledge that OnVoya has no express or implied obligation to announce or make available a commercial version of the Pre-Release Services to anyone in the future. Should a commercial version be made available, it may have features or functionality that are different from those found in the Pre-Release Services.
    </div>
    <div
      className="subti">
      Disclaimer of Warranties
    </div>
    <div
      className="sub-copy caps">
      THE ONVOYA SERVICE AND ANY PRE-RELEASE SERVICES ARE OFFERED \"AS IS\". THAT MEANS WE DON\'T GUARANTEE THE QUALITY, QUANTITY, COMPLETENESS, ACCURACY, AVAILABILITY, OR SPEED OF THE ONVOYA SERVICE AND ANY PRE-RELEASE SERVICES, OR MAKE ANY OTHER REPRESENTATION ABOUT THE INFORMATION WE DISPLAY, INCLUDING BUT NOT LIMITED TO: PRICES, SCHEDULES, FARES, PHOTOGRAPHS, SEAT AVAILABILITY, AIRCRAFT TYPES, AIRLINE NAMES, FREQUENT FLYER MILE EARNING, HOTEL AMENITIES, HOTEL RATINGS, OR THE HELP WE PROVIDE VIA LIVE CHAT OR EMAIL.
    </div>
    <div
      className="sub-copy caps">
      ONVOYA AND ITS LICENSORS AND SUPPLIERS HEREBY DISCLAIM ALL WARRANTIES OF ANY KIND–EXPRESS OR IMPLIED–INCLUDING, BUT NOT LIMITED TO, THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, TITLE, AND NON-INFRINGEMENT. ONVOYA DISCLAIMS ALL WARRANTIES AND CONDITIONS THAT THE SERVICE OR ANY EMAIL SENT FROM ONVOYA IS FREE OF VIRUSES OR OTHER HARMFUL COMPONENTS.
    </div>
    <div
      className="sub-copy caps">
      NO INFORMATION OR ADVICE, WHETHER ORAL OR WRITTEN, OBTAINED BY YOU FROM ONVOYA WILL CREATE ANY WARRANTY THAT IS NOT EXPRESSLY STATED IN THESE TERMS. SOME JURISDICTIONS DO NOT ALLOW DISCLAIMER OF WARRANTIES AND YOU MAY HAVE OTHER RIGHTS THAT VARY BY JURISDICTION. IN SUCH JURISDICTIONS, THE FOREGOING DISCLAIMERS SHALL APPLY TO THE FULLEST EXTENT PERMITTED BY LAW.
    </div>
    <div
      className="subti">
      Limitation of Liability
    </div>
    <div
      className="sub-copy caps">
      IN EXCHANGE FOR ONVOYA\'S PROVISION OF THE ONVOYA SERVICE TO YOU, YOU AGREE THAT ONVOYA IS ENTITLED TO A LIMITATION OF LIABILITY. WHAT THAT MEANS IS THAT IN NO EVENT WILL ONVOYA, OR ITS LICENSORS OR SUPPLIERS, BE LIABLE ON ANY CLAIMS ARISING OUT OF OR RELATING TO YOUR USE OF OR INABILITY TO USE THE ONVOYA SERVICE, ON ANY THEORY, INCLUDING BUT NOT LIMITED TO NEGLIGENCE, CONTRACT, TORT, STRICT LIABILITY, OR OTHERWISE. THIS LIMITATION OF LIABILITY APPLIES TO, BUT SHALL NOT BE LIMITED TO ANY CLAIM FOR: (I) DIRECT, INDIRECT, SPECIAL, INCIDENTAL, CONSEQUENTIAL, OR EXEMPLARY DAMAGES, SUCH AS DAMAGES FOR LOSS OF PROFITS, GOODWILL, OR OTHER INTANGIBLE LOSSES (EVEN IF ONVOYA OR ANY SUPPLIER OR LICENSOR HAS BEEN ADVISED OF THE POSSIBILITY OF THESE DAMAGES); (II) THE COST OF PROCUREMENT OF SUBSTITUTE PRODUCTS OR SERVICES; (III) INTERRUPTION OF USE OR LOSS OR CORRUPTION OF DATA; OR (IV) ANY AMOUNTS PAID BY YOU TO ANY OF ONVOYA\'S BUSINESS PARTNERS, INCLUDING BUT NOT LIMITED TO OUR DATA PROVIDERS AND THE SITES TO WHICH WE LINK. IN NO EVENT WILL ONVOYA BE LIABLE TO YOU FOR ANY MORE THAN THE TOTAL AMOUNTS YOU HAVE PAID TO ONVOYA.
    </div>
    <div
      className="sub-copy caps">
      THIS LIMITATION OF LIABILITY SHALL ALSO APPLY WITH RESPECT TO DAMAGES INCURRED BY REASON OF OTHER SERVICES OR PRODUCTS RECEIVED THROUGH OR ADVERTISED ON THE ONVOYA SERVICE, OR ADVICE RECEIVED THROUGH OR ADVERTISED ON THE ONVOYA SERVICE, OR SERVICES, PRODUCTS OR ADVICE RECEIVED THROUGH ANY LINKS PROVIDED IN THE ONVOYA SERVICE. (OF PARTICULAR IMPORTANCE, THIS MEANS WE ARE NOT RESPONSIBLE IF YOUR FLIGHT, TRAIN, OR OTHER TRAVEL IS DELAYED OR CANCELED, OR IF YOUR HOTEL RESERVATION IS ALTERED OR CANCELED.) THE INCLUSION, RECEPTION, OFFERING, OR ADVERTISING OF ANY SERVICES OR PRODUCTS THROUGH OR ON THE ONVOYA SERVICE DOES NOT CONSTITUTE ANY ENDORSEMENT OR RECOMMENDATION OF SUCH SERVICES OR PRODUCTS BY ONVOYA.
    </div>
    <div
      className="sub-copy caps">
      EACH OF THESE PROVISIONS IS SEVERABLE AND INDEPENDENT OF ALL OTHER PROVISIONS OF THESE TERMS. THE LIMITATIONS IN THIS SECTION WILL APPLY EVEN IF ANY LIMITED REMEDY FAILS OF ITS ESSENTIAL PURPOSE. THE FOREGOING SHALL NOT APPLY TO THE EXTENT PROHIBITED BY APPLICABLE LAW.
    </div>
    <div
      className="sub-copy">
      The OnVoya Service is controlled and offered by OnVoya in the United States of America. OnVoya makes no representations that the OnVoya Service is appropriate or available for use in other locations. Those who access or use the OnVoya Service from other jurisdictions do so at their own volition and are responsible for compliance with local law.
    </div>
    <div
      className="subti">
      Third Party Content and Services
    </div>
    <div
      className="sub-copy">
      As you may notice, the OnVoya Service may contain links to third party products, services and/or websites. We don\'t have any control over these third parties, and we are not responsible for their performance. We also don\'t endorse them, and are not responsible for their terms, their privacy, data security, or other practices, or their content, advertising, or other materials they may provide to you. Your security when using these third parties is your responsibility.
    </div>
    <div
      className="sub-copy">
      You may integrate various functionalities from other services, such as Microsoft Outlook and Google Calendar, with the OnVoya service. If you choose to integrate another service\'s functionality with OnVoya, you understand that that service\'s terms and conditions are what govern your relationship with that service, not these Terms of Service.
    </div>
    <div
      className="sub-copy">
      You also may interact with the OnVoya Virtual Travel Agent via a third party service, such as Facebook Messenger or Slack. Again, you understand that your relationships with those third-party services are governed by the terms of service for those services, not these Terms of Service, and that we are not responsible for these third parties\' privacy, data security, or other practices, or their content, advertising, or other materials they may provide to you.
    </div>
    <div
      className="subti">
      Booking Travel Through Our Travel Partners
    </div>
    <div
      className="sub-copy">
      OnVoya offers users the ability to book travel with some of our third-party travel partners through the OnVoya Service. We may require you to create an OnVoya account or connect to the OnVoya Service through your Facebook, Google or Twitter account in order to book travel through one of our travel partners. If you use a booking feature that\'s powered by one of our travel partners, we collect your payment information and pass it on to the travel partner to process the transaction, consistent with our Privacy Policy. We don\'t complete the transaction or process your payment–the travel partner does. You understand and agree that if you encounter any problems with the transaction or with the reservation you book in that transaction, you must resolve your dispute with the travel partner, not with OnVoya, and your sole remedy, including any refund, lies with the travel partner, not with OnVoya.  However, if you cannot reach a satisfactory resolution with one of our travel partners, we want to know and we may, at our sole discretion, intervene on your behalf.
    </div>
    <div
      className="sub-copy">
      Additional terms, provided by the travel partner, apply to your transaction or other use of the booking feature powered by that partner, and you acknowledge and agree to be bound by those additional terms.
    </div>
    <div
      className="sub-copy">
      You understand that the applicable partner\'s terms govern your booking transaction, and that in the event of any material conflict between those terms and this Agreement with respect to your booking transaction or other use of the booking feature, those terms shall prevail. OnVoya has no control over its third-party travel partners and is not responsible for their terms, their privacy, data security or other practices, their performance, or their content, advertising, or other materials they may provide to you, whether or not provided through the Service.
    </div>
    <div
      className="subti">
      Modifications to the OnVoya Service
    </div>
    <div
      className="sub-copy">
      We\'re still working on the OnVoya service, so from time to time we may modify, discontinue, or restrict, temporarily or permanently, portions or all of the OnVoya service. We may do this without notifying you. We reserve our right to make these changes for any reason or no reason at all. We also reserve our right to terminate your license to use the OnVoya service, and to block, restrict or prevent your future access to or use of the service. Again, we may do this without notice to you, and for any or no reason. Neither we nor our suppliers or licensors will be liable to you or to any third party for any modification, discontinuance or restriction of the OnVoya service.
    </div>
    <div
      className="subti">
      Customer Service
    </div>
    <div
      className="sub-copy">
      If you have any problems using the OnVoya service, we\'re sorry! Our customer support representatives at
      <strong>
        support@OnVoya.com
      </strong>
       are ready to assist you. To help resolve your issue, the customer service representative who\'s assisting you may log in to the OnVoya service under your username, but only with your permission and only for the duration of the customer service session. Customer service representatives don\'t know your account password, and you don\'t need to provide it in order for them to help you. You may also choose to contact our Virtual Travel Agent through a third-party messenger service to receive immediate responses to your questions from our Virtual Travel Agent.
    </div>
    <div
      className="subti">
      Your Privacy
    </div>
    <div
      className="sub-copy">
      We may collect registration, payment, and other information about you through the OnVoya Service. Our collection and use of this information is described in the OnVoya 
      <a
        onClick={[Function]}
        style={Object {}}>
        <u>
          Privacy Policy.
        </u>
      </a>
    </div>
    <div
      className="subti">
      Ability to Accept Terms of Service
    </div>
    <div
      className="sub-copy">
      You affirm that you are either at least 18 years of age, or an emancipated minor, or possess legal parental or guardian consent, and are fully able and competent to enter into the terms, conditions, obligations, affirmations, representations, and warranties set forth in these Terms of Service, and to abide by and comply with these Terms of Service. OnVoya is not intended for or directed to persons under the age of 13. Any person who provides their information to OnVoya through any part of the OnVoya Service represents to OnVoya that he or she is 13 years of age or older. If you are under the age of 13, then please do not use the OnVoya Service.
    </div>
    <div
      className="subti">
      Termination
    </div>
    <div
      className="sub-copy">
      These Terms of Service will continue to apply to you until either you or OnVoya terminates this Agreement. If you connect to the OnVoya Service using your Facebook, Google or Twitter account, you may terminate this Agreement at any time by disconnecting your account from the OnVoya Service and no longer using the Service thereafter. If you have created a OnVoya account, you may terminate this Agreement at any time by sending a notice confirming such termination to OnVoya by email to 
      <strong>
        contacts@OnVoya.com.
      </strong>
    </div>
    <div
      className="sub-copy">
      OnVoya may suspend your access to the OnVoya Service and/or terminate this Agreement at any time, with or without notice to you, (i) if you are in breach of any other provision of these Terms of Use or if you make unauthorized use of the OnVoya Service; (ii) if OnVoya elects at its discretion to cease providing access to the OnVoya Service in the jurisdiction where you reside or from where you are attempting to access the OnVoya Service, or (iii) in other reasonable circumstances as determined by OnVoya at its discretion. The provisions of these Terms of Use that are intended by their nature to survive the termination of the Agreement will survive the termination of the Agreement, including, but not limited to, those Sections entitled Your Responsibilities, Intellectual Property, Disclaimer of Warranties, Limitation of Liability, Payments, Privacy Policy, Integration and Severability, Governing Law, Arbitration, and Arbitration Agreement, respectively. If you or OnVoya terminate this Agreement, or if OnVoya suspends your access to the OnVoya Service, you agree that OnVoya shall have no liability or responsibility to you and OnVoya will not refund any amounts that you have already paid, to the fullest extent permitted under applicable law, except as otherwise provided under these Terms.
    </div>
    <div
      className="subti">
      Integration and Severability
    </div>
    <div
      className="sub-copy">
      These Terms of Use, together with the OnVoya Privacy Policy (which is hereby incorporated by this reference), constitute the entire agreement between you and OnVoya with respect to your use of the OnVoya Service, and supersede any prior agreement between you and OnVoya. Should one or more provisions of these Terms of Use be found to be unlawful, void or unenforceable, such provision(s) shall be deemed severable and will not affect the validity and/or enforceability of the remaining provisions of the Terms of Use, which will therefore remain in full force and effect. Our failure or delay in enforcing any provision of this Agreement at any time does not waive our right to enforce the same or any other provision(s) of the Agreement in the future. OnVoya reserves any rights not expressly granted herein.
    </div>
    <div
      className="subti">
      Governing Law
    </div>
    <div
      className="sub-copy">
      This Agreement, and all legal issues arising from or related to the OnVoya Service, shall be governed by and construed in accordance with the laws of the State of California without regard to that state\'s conflict of law provisions.
    </div>
    <div
      className="subti">
      Arbitration
    </div>
    <div
      className="sub-copy">
      All disputes between you and OnVoya will be resolved by BINDING ARBITRATION as described in more detail in the Arbitration Agreement included below. You thus GIVE UP YOUR RIGHT TO GO TO COURT to assert or defend your rights under this agreement (except for matters that may be taken to small claims court). Your rights will be determined by a NEUTRAL ARBITRATOR and NOT a judge or jury and your claims cannot be brought as a class action. You are entitled to a fair hearing, but the arbitration procedures are simpler and more limited than they would be in court. Arbitrator decisions are as enforceable as any court order and are subject to very limited review by courts or judges.
    </div>
    <div
      className="subti">
      Arbitration Agreement
    </div>
    <div
      className="sub-copy">
      You and OnVoya agree that any dispute, claim or controversy arising out of or relating in any way to the OnVoya Service, our website or user interfaces, or these Terms of Use shall be determined by binding arbitration or small claims court instead of in courts of general jurisdiction. You and OnVoya agree that this arbitration agreement extends to the determination of the scope or applicability of this agreement to arbitrate. Arbitration is more informal than a lawsuit in court. Arbitration uses a neutral arbitrator instead of a judge or jury, allows for more limited discovery than in court, and is subject to very limited review by courts. Arbitrators can award the same damages and relief that a court can award. Any arbitration under this Agreement will take place on an individual basis; class arbitrations and class actions are not permitted.
    </div>
    <div
      className="sub-copy">
      You agree that, by entering into this Agreement, you and OnVoya are each waiving the right to a trial by jury or to participate in a class action. The Federal Arbitration Act governs the interpretation and enforcement of this provision. This arbitration provision shall survive termination of this Agreement and the termination of your use of the OnVoya Service.
    </div>
    <div
      className="sub-copy">
      If you elect to seek arbitration, you must first send to OnVoya, by certified mail, a written Notice of Dispute (\"Notice\"). The Notice to OnVoya should be addressed to: OnVoya, Attn: Legal, 592 Dory Ln., Redwood Shores, CA 94065 (\"Notice Address\"). The Notice must (a) describe the nature and basis of the claim or dispute; and (b) set forth the specific relief sought (\"Demand\"). If OnVoya and you do not reach an agreement to resolve the claim within 30 days after the Notice is received, you or OnVoya may commence an arbitration proceeding or file a claim in small claims court. During the arbitration, the amount of any settlement offer made by OnVoya or you shall not be disclosed to the arbitrator until after the arbitrator determines the amount, if any, to which you or OnVoya is entitled. You may download or copy a form Notice and a form to initiate arbitration at www.adr.org.
    </div>
    <div
      className="sub-copy">
      After OnVoya receives notice at the Notice Address that you have commenced arbitration, it will promptly reimburse you for your payment of the filing fee, unless your claim is for greater than $10,000. (The filing fee currently is $125 for claims under $10,000 but is subject to change by the arbitration provider. If you are unable to pay this fee, OnVoya will pay it directly upon receiving a written request at the Notice Address.) The arbitration will be governed by the Commercial Arbitration Rules and the Supplementary Procedures for Consumer Related Disputes (collectively, \"AAA Rules\") of the American Arbitration Association (\"AAA\"), as modified by this Agreement, and will be administered by the AAA.
    </div>
    <div
      className="sub-copy">
      The AAA Rules are available online at www.adr.org, by calling the AAA at 1-800-778-7879, or by writing to the Notice Address. The arbitrator is bound by the terms of this Agreement. All issues are for the arbitrator to decide, including issues relating to the scope and enforceability of this arbitration agreement. Unless OnVoya and you agree otherwise, any arbitration hearings will take place in the county (or parish) of your primary residence. If your claim is for $10,000 or less, we agree that you may choose whether the arbitration will be conducted solely on the basis of documents submitted to the arbitrator, through a telephonic hearing, or by an in-person hearing as established by the AAA Rules. If your claim exceeds $10,000, the right to a hearing will be determined by the AAA Rules. Regardless of the manner in which the arbitration is conducted, the arbitrator shall issue a reasoned written decision sufficient to explain the essential findings and conclusions on which the award is based. Except as otherwise provided for herein, OnVoya will pay all AAA filing, administration, and arbitrator fees for any arbitration initiated in accordance with the notice requirements above.
    </div>
    <div
      className="sub-copy">
      If, however, the arbitrator finds in favor of OnVoya as to the substance of your claim or the relief sought, you will reimburse OnVoya for your portion of the filing fee. If the arbitrator finds that either the substance of your claim or the relief sought in the Demand is frivolous or brought for an improper purpose (as measured by the standards set forth in Federal Rule of Civil Procedure 11(b)), then the payment of all AAA filing, administration, and arbitrator fees will be governed by the AAA Rules. In such case, you agree to reimburse OnVoya for all monies previously disbursed by it that are otherwise your obligation to pay under the AAA Rules. In addition, if you initiate an arbitration in which you seek more than $10,000 in damages, the payment of these fees will be governed by the AAA rules. The arbitrator may award declaratory or injunctive relief only in favor of the individual party seeking relief and only to the extent necessary to provide relief warranted by that party\'s individual claim.
    </div>
    <div
      className="sub-copy">
      YOU AND ONVOYA AGREE THAT EACH MAY BRING CLAIMS AGAINST THE OTHER ONLY IN YOUR OR ITS INDIVIDUAL CAPACITY, AND NOT AS A PLAINTIFF OR CLASS MEMBER IN ANY PURPORTED CLASS OR REPRESENTATIVE PROCEEDING. Further, unless both you and OnVoya agree otherwise, the arbitrator may not consolidate more than one person\'s claims with your claims, and may not otherwise preside over any form of a representative or class proceeding. If this specific provision is found to be unenforceable, then the entirety of this arbitration provision shall be null and void.
    </div>
    <div
      className="sub-copy">
      Notwithstanding any provision in this Agreement to the contrary, we agree that if OnVoya makes any future change to this arbitration provision (other than a change to the Notice Address), you may reject any such change by sending us written notice within 30 days of the change to the Arbitration Notice Address provided above. By rejecting any such change, you are agreeing that you will arbitrate any dispute between us in accordance with the language of this provision, and agree to promptly cease using the OnVoya Service.
    </div>
    <div
      className="subti">
      Mobile Terms
    </div>
    <div
      className="sub-copy">
      If you use OnVoya\'s mobile apps to access the OnVoya Service (the \"Mobile Service\"), you accept and agree to be bound by the terms in this section, in addition to the rest of these Terms of Service. Your continued use of the Mobile Service is conditioned upon your compliance with these Mobile Terms.
    </div>
    <div
      className="sub-copy">
      By using the Mobile Service, you understand and agree that OnVoya has no obligation to maintain, support, upgrade, or update the Mobile Service, or to provide all or any specific content through the Mobile Service. You further understand and agree that we may access, collect, and use information, which may include personally identifiable information, which your mobile device automatically makes available to our mobile app, consistent with our 
      <a
        onClick={[Function]}
        style={Object {}}>
        <u>
          Privacy Policy
        </u>
      </a>
      .
    </div>
  </div>
   
</div>
`;
